subprojects/extTest/src/test/java/org/mockitousage/plugins/switcher/MyPluginSwitch.java

```diff
@@ -14,6 +14,10 @@ public class MyPluginSwitch implements PluginSwitch {
     static List<String> invokedFor = new LinkedList<String>();
 
     public boolean isEnabled(String pluginClassName) {
+        if (!pluginClassName.startsWith("org.mockitousage")) {
```

Comment:
Filtering the plugins from the envinronment (like `member-accessor-inline`, `mock-maker-inline`, ...)